### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by 4812571**
Integrated Mayhem, an automated fuzz-testing solution.

### CODE REVIEW

File: .github/workflows/mayhem.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: Mayhem
```
Comment (xansec) [inline .github/workflows/mayhem.yml:1]: Please use the latest workflow version.

File: fuzz/fuzz_targets/format_code.rs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#![no_main]
```
Comment (xansec) [inline fuzz/fuzz_targets/format_code.rs]: Is this missing some code?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: fuzz/fuzz_targets/format_code.rs (hunk 2 above)
Comment (4812571) [reply fuzz/fuzz_targets/format_code.rs]: No, my apologies, That was vestigial code from before I separated it into two targets.

File: .github/workflows/mayhem.yml (hunk 1 above)
Comment (4812571) [reply .github/workflows/mayhem.yml:1]: Updated!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**